src/util/splitSentence.ts

```diff
@@ -55,26 +55,21 @@ const splitSentence = (value: string): string[] => {
   // pattern2, multiple symbols: ?! !!! ...
   const mainSplitRegex = /[.;!?]+/g
 
-  const splitters = value.match(mainSplitRegex)
-
-  // When it cannot be split by the main spliter, spliter by ','
-  if (!splitters)
-    return value
-      .split(',')
-      .filter(s => s !== '')
-      .map(s => s.trim())
+  const sentenceSplitters = value.match(mainSplitRegex)
 
   /**
-   * Checks if the value has no other main split characters  except one period at the end.
+   * Checks if the value has no other main split characters  except one period at the end, i.e. value is just one sentence.
    * If so, allow split on comma only if there are no main split characters in the value or has only one period at the end.
    */
   const hasOnlyPeriodAtEnd = once(() => /^[^.;!?]*\.$[^.;!?]*/.test(value.trim()))
 
-  if (hasOnlyPeriodAtEnd())
+  // if we're sub-sentence or in one sentence territory, split by comma and "and"
+  // e.g. "john, johnson, and john doe" -> "- john - johnson - john doe"
+  if (!sentenceSplitters || hasOnlyPeriodAtEnd())
     return value
-      .split(',')
+      .split(/,|and/i)
+      .map(s => s.trim())
       .filter(s => s !== '')
```

Comment:
swapping those allows me to have a simple regex that doesn't care about space

this commit shows that we either do a more complex regex with `\s*` in it, or just trim first, then filter :D
imo, this is way more concise
https://github.com/cybersemics/em/pull/2658/commits/36d59071a6e1ed0a55afbac15bdce07696491399